README.md

```diff
@@ -15,6 +15,15 @@ KubeArmor leverages [Linux security modules \(LSMs\)](https://en.wikipedia.org/w
 
 ![KubeArmor High Level Design](.gitbook/assets/kubearmor_overview.png)
 
+## Use Cases
+### Kubernetes Runtime Security
+Kubearmor provides runtime protection for the Kubernetes workloads. It can restrict the runtime behavior of the pods/containers running on the k8s node at the system level using the Linux Security Modules. Apparmor, SElinux and BPF LSM are currently supported LSMs by Kubearmor. It Provides easy-to-use Policy semantics using Kubernetes CRDs. K8s custom resource `KubeArmorPolicy` can be used by the user to specify the security policy for the container/pod. 
```

Comment:
Kubearmor provides runtime protection for the Kubernetes workloads by restricting the runtime behavior of the pods/containers at the system level using Linux Security Modules (LSMs). KubeArmor currently supports BPF-LSM, AppArmor and SELinux. It provides easy-to-use policy semantics using Kubernetes CRD - `KubeArmorPolicy` - which can be used to specify the security policy.